src/test/java/com/google/api/codegen/testdata/java/java_library.baseline

```diff
@@ -7651,6 +7749,10 @@ public class LibraryClientTest {
     Assert.assertEquals(optionalRepeatedFixed32, actualRequest.getOptionalRepeatedFixed32List());
     Assert.assertEquals(optionalRepeatedFixed64, actualRequest.getOptionalRepeatedFixed64List());
     Assert.assertEquals(optionalMap, actualRequest.getOptionalMapMap());
+    Assert.assertTrue(
+        channelProvider.isHeaderSent(
+            ApiClientHeaderProvider.getDefaultApiClientHeaderKey(),
+            GaxGrpcProperties.getDefaultApiClientHeaderPattern()));
```

Comment:
Do the generated tests pass on api-client-staging?

Reply:
Our api-client-staging for java is outdated (broken?). But yes, it works (run artman against latest toolkit & gax fro langauge and commons, then run ./gradlew clean test (excluding others, to make things faster) and it works).